python/lsst/jointcal/jointcal.py

```diff
@@ -216,22 +230,23 @@ def run(self, ref, tract):
         chi2 = fit.ComputeChi2()
         print(chi2)
 
-        for i in range(20) :
-            r = fit.Minimize("Distortions Positions",5) # outliers removal at 5 sigma.
+        for i in range(20):
+            r = fit.Minimize("Distortions Positions", 5)  # outliers removal at 5 sigma.
```

Comment:
I think it's saying "minimize both of these quantities", but I'm not going to touch it for now.